src/api/app/assets/stylesheets/webui2/watchlist.scss

```diff
@@ -0,0 +1,7 @@
+#global-navigation {
+  ul {
+    .dropdown-menu.dropdown-menu-right {
+      z-index: 1021;
```

Comment:
I wonder, why 1021?

Reply:
@dmarcoux the toolbar z-index is set to auto but it seems it is taking 1020 by default. Probaby following this Bootstrap configuration:

```$zindex-sticky:            1020 !default;```
https://getbootstrap.com/docs/4.1/layout/overview/#z-index

Therefore, 1021 is the first value that shows the dropdown menu above the toolbar.

See also: #6538